conditions-and-loops/conditions1.js

```diff
@@ -0,0 +1,21 @@
+const averageGrade = (Math.random() * 100).toFixed();
```

Comment:
averageGrade is type of string, convert it to number

Reply:
comparison operators transform `averageGrade` to string automatically, and in this example `averageGrade` is an integer, so it's completely safe
https://javascript.info/comparison#comparison-of-different-types